tst.c

```diff
@@ -222,7 +222,7 @@ void *tst_ins_del(tst_node **root, const char *s, const int del, const int cpy)
          * "Conditional jump or move depends on uninitialised value(s)"
          */
         if (!(*pcurr = calloc(1, sizeof **pcurr))) {
-            fprintf(stderr, "error: tst_insert(), memory exhausted.\n");
```

Comment:
Don't touch this file.

Reply:
Always minimize the scope of files you are about to change.

Reply:
Sorry, I modified the wrong file